modules/generator/src/main/java/org/lwjgl/generator/util/ModuleInfoGen.java

```diff
@@ -67,15 +67,15 @@ public static void main(String[] args) throws IOException {
         }
 
         @Override public boolean equals(Object other) {
-            return name.equals(((Module)other).name);
+            return other != null && (other instanceof Module) && Objects.equals(name, ((Module)other).name);
```

Comment:
`other != null` is redundant with `other instanceof Module`, which already returns `false` when `other` is `null`.

Reply:
exactly.